plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryUtil.java

```diff
@@ -66,4 +68,9 @@ public static String toBigQueryColumnName(String columnName)
         }
         return columnName;
     }
+
+    public static boolean isWildcardTable(TableDefinition.Type type, String tableName)
+    {
+        return type == TABLE && tableName.contains("*");
```

Comment:
Before the change, could a user create a table with asterisk in its name?

Reply:
No. Table name must be alphanumeric and underscores.